app/models/katello/glue/pulp/node.rb

```diff
@@ -0,0 +1,23 @@
+module Katello
+  module Glue::Pulp::Node
```

Comment:
There is a team working on removing the Pulp node concept and replacing it with direct calls to the Pulp on a capsule, you may want to coordinate this with them @jlsherrill @johnpmitsch @cfouant